### PR TITLE
v5.25.5 and v5.26.2 dot-release updates

### PR DESCRIPTION
**Pull request by lindalumitchell**
Changelog, Version Archive, ESR page updates for the v5.25.5 and v5.26.2 dot release.

Edit: Added Upgrade Notes page.

Only one review is required to merge.

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
 ## Release v5.26 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
 
+- **v5.26.2, released 2020-09-03**
+  - Forcefully disables the SAML Setting "Use Improved SAML Library (Beta)”, as we have identified some issues in this feature. Please follow instructions at https://docs.mattermost.com/deployment/sso-saml-before-you-begin.html for enabling SAML using the feature-equivalent xmlsec1 utility.
```
Comment (lieut-data) [inline source/administration/changelog.md:10]: Nit: might consider wrapping `xmlsec1` in backticks.

File: source/administration/extended-support-release.rst [hunk 2]
```diff
@@ -44,7 +44,7 @@ What are the current supported Extended Support Release versions?
 +-------------+----------------+------------------+------------------+--------------------------------------------------------------------------------------------+-------------------------------------------------+
 | Version     | Type           | Release Date     | End of Support   | Latest Dot Release Download link                                                           | Upgrade Notes                                   |
 +=============+================+==================+==================+============================================================================================+=================================================+
-| 5.25        | Quality        | July 16, 2020    | April 15, 2021   | `5.25.4 <https://releases.mattermost.com/5.25.4/mattermost-5.25.4-linux-amd64.tar.gz>`_    |                                                 |
+| 5.25        | Quality        | July 16, 2020    | April 15, 2021   | `5.25.5 <https://releases.mattermost.com/5.25.5/mattermost-5.25.5-linux-amd64.tar.gz>`_    |                                                 |
```
Comment (lieut-data) [inline source/administration/extended-support-release.rst:47]: Should we call out upgrade notes here, given this is effectively a breaking change?

**Review by lieut-data — APPROVED** [review]

File: source/administration/extended-support-release.rst (hunk 2 above)
Comment (lindalumitchell) [reply source/administration/extended-support-release.rst]: Thanks @lieut-data, upgrade notes slipped off my radar. Added a note on this page, and also added the Upgrade Notes doc to this PR. Followed @wiersgallak's suggestion in-channel to add a note on that page to all versions after v5.20.

Re-requesting review, if you wouldn't mind taking a look at the upgrade notes as well?

**Review by lieut-data — APPROVED** [review]

File: source/administration/extended-support-release.rst [hunk 3]
```diff
@@ -44,7 +44,9 @@ What are the current supported Extended Support Release versions?
...
 +=============+================+==================+==================+============================================================================================+=================================================+
-| 5.25        | Quality        | July 16, 2020    | April 15, 2021   | `5.25.4 <https://releases.mattermost.com/5.25.4/mattermost-5.25.4-linux-amd64.tar.gz>`_    |                                                 |
+| 5.25        | Quality        | July 16, 2020    | April 15, 2021   | `5.25.5 <https://releases.mattermost.com/5.25.5/mattermost-5.25.5-linux-amd64.tar.gz>`_    | SAML Setting "Use Improved SAML                 |
+|             |                |                  |                  |                                                                                            | Library (Beta)" was disabled.                   |
+|             |                |                  |                  |                                                                                            | Please use ``xmlsec1``.                         |
```
Comment (jasonblais) [inline source/administration/extended-support-release.rst:49]: Propose

```
Please use ``xmlsec1`` utility <https://docs.mattermost.com/deployment/sso-saml-before-you-begin.html>`_.
```

with a link to the instructions.

**Review by jasonblais — APPROVED** [review]
One proposal below @lindalumitchell, non-blocking.

File: source/administration/extended-support-release.rst (hunk 3 above)
Comment (lindalumitchell) [reply source/administration/extended-support-release.rst:49]: I missed this comment in the urgency to merge the updates, sorry about that.

**Pull request merged**